lib/cli/package.json

```diff
@@ -42,33 +42,6 @@
     "shelljs": "^0.8.2",
     "update-notifier": "^2.5.0"
   },
-  "devDependencies": {
```

Comment:
Had to remove these dependencies to make it work.

I makes sense NOT not to have these dependencies, because the files using these will never be run from inside our codebase, they are merely a template.